app/models/planned_disbursement.rb

```diff
@@ -2,6 +2,14 @@ class PlannedDisbursement < ApplicationRecord
   include PublicActivity::Common
   include HasFinancialQuarter
 
+  DIRECT_ACCESS_WARNING = "The PlannedDisbursement model should not be accessed \
+directly; please use the PlannedDisbursementHistory and PlannedDisbursementOverview \
+services to create forecasts. See doc/forecasts-and-versioning.md for more information."
+
+  default_scope do
```

Comment:
I never thought I would see an safe use for `default_scope` but this definately fits the bill!